pyomo/contrib/pyros/separation_problem_methods.py

```diff
@@ -419,68 +500,139 @@ def initialize_separation(model_data, config):
             v.fix()
 
     if any(c.active for c in model_data.separation_model.util.h_x_q_constraints):
-        raise AttributeError("All h(x,q) type constraints must be deactivated in separation.")
+        raise AttributeError(
+            "All h(x,q) type constraints must be deactivated in separation."
+        )
+
+    # check: initial point feasible?
+    for con in sep_model.component_data_objects(Constraint, active=True):
+        lb, val, ub = value(con.lb), value(con.body), value(con.ub)
+        lb_viol = val < lb - 1e-5 if lb is not None else False
```

Comment:
There's a hard-coded tolerance here of 1e-5...I assume that was agreed upon at some point, but should it be made into a global variable somewhere so we don't loose track of it? Like `initial_point_feas_tol` or something? Also out of curiosity, why 1e-5?

Reply:
My choice of `1E-5` is somewhat arbitrary; I chose this initially as this is the default absolute constraint violation tolerance for BARON, a solver I use frequently with PyROS. I could make this a global variable (perhaps defined in `util.py`), or opt for `config.robust_feasibility_tolerance` instead.